app/router.jsx

```diff
@@ -95,9 +95,8 @@ class ExternalRedirect extends React.Component {
 
 export const routes = (
   <Route path="/" component={require('./partials/app')}>
-    <IndexRoute component={HomePageRoot} />
+    <IndexRoute component={() => <ExternalRedirect newUrl='https://www.zooniverse.org' />} />
     <Route path="home" component={ONE_UP_REDIRECT} />
-    <Route path="home-for-user" component={require('./pages/home-for-user').default} />
```

Comment:
These changes are the most consequential. There has to be a `<Route path="/"` importing `'./partials/app'` in order for PFE to work at all. However, the ExternalRedirect in IndexRoute does the redirecting when a browser hits `/`. 

I also kept `path="home"` because I'm unsure of its legacy, but if you hit /home you'll be redirected to the static proxy (and on to FEM). It was mentioned here: https://github.com/zooniverse/Panoptes-Front-End/pull/6041